src/uu/nproc/src/nproc.rs

```diff
@@ -25,7 +25,9 @@ pub const _SC_NPROCESSORS_CONF: libc::c_int = 1001;
 static OPT_ALL: &str = "all";
 static OPT_IGNORE: &str = "ignore";
 
-static ABOUT: &str = "Print the number of cores available to the current process.";
+static ABOUT: &str = r#"Print the number of cores available to the current process.\
+If the OMP_NUM_THREADS or OMP_THREAD_LIMIT environment variables are set, then\
+they will determine the minimum and maximum returned value respectively."#;
```

Comment:
I think the `r#...#` should be removed here, otherwise the `\` don't work. The output is now:
```
nproc 0.0.12
Print the number of cores available to the current process.\
If the OMP_NUM_THREADS or OMP_THREAD_LIMIT environment variables are set, then\
they will determine the minimum and maximum returned value respectively.
```
If I'm not mistaken there should be a space before the `\` so that it won't print `process.If` without a space inbetween